src/Meziantou.Framework.FastEnumToStringGenerator/EnumToStringSourceGenerator.cs

```diff
@@ -146,9 +148,19 @@ private static string GenerateCode(ImmutableArray<EnumToProcess> enums)
                     sb.AppendLine("{");
                 }
 
+                if (typeIsPublic)
```

Comment:
I don't think you can add the comment on all types, even internal ones. This adds value as the xml comment is visible in the IDE.

Reply:
It depends as the comment does not add much value but rather satisfies the compiler 😃